src/components/Projects/index.tsx

```diff
@@ -1,36 +1,56 @@
 import { InfoCircleOutlined } from '@ant-design/icons'
 import { t, Trans } from '@lingui/macro'
-import { Button, Checkbox, Select, Space, Tooltip } from 'antd'
+import { Button } from 'antd'
 import Search from 'antd/lib/input/Search'
 import FeedbackFormLink from 'components/shared/FeedbackFormLink'
 import Loading from 'components/shared/Loading'
-import ProjectsGrid from 'components/shared/ProjectsGrid'
 
 import { ThemeContext } from 'contexts/themeContext'
+
 import { useInfiniteProjectsQuery, useProjectsSearch } from 'hooks/Projects'
-import { ProjectState } from 'models/project-visibility'
+import { ProjectCategory, ProjectStateFilter } from 'models/project-visibility'
 import { TerminalVersion } from 'models/terminal-version'
-import { useContext, useEffect, useMemo, useRef, useState } from 'react'
+import React, { useContext, useEffect, useMemo, useRef, useState } from 'react'
 
-import { layouts } from 'constants/styles/layouts'
+import Grid from 'components/shared/Grid'
+import ProjectCard from 'components/shared/ProjectCard'
 
+import { layouts } from 'constants/styles/layouts'
+import TrendingProjects from './TrendingProjects'
+import ProjectsTabs from './ProjectsTabs'
+import MyProjects from './MyProjects'
+import ProjectsFilterAndSort from './ProjectsFilterAndSort'
+import ArchivedProjectsMessage from './ArchivedProjectsMessage'
 type OrderByOption = 'createdAt' | 'totalPaid'
 
 const pageSize = 20
 
 export default function Projects() {
+  // Checks if user came from homepage trending section,
+  // in which case auto open trending tab
+  const trendingTabOpen = window.location.hash.split('=')[1] === 'trending'
```

Comment:
**suggestion**: it's probably preferrable to use react router `useParams` hook:

https://v5.reactrouter.com/web/api/Hooks/useparams

Reply:
I was trying to use `useSearchParams` for this, but for some reason the search parametre was empty. The only way I could get query string was with .hash()

<img width="666" alt="Screen Shot 2022-02-01 at 11 57 51 am" src="https://user-images.githubusercontent.com/96150256/151902390-c43c22f1-cdb7-4d78-af12-b1de9b7ede8f.png">

Reply:
ah, shit. Its probably because we have the `/#/` in the path :/

Reply:
Try this @johnnyd-eth 

```js
  const location = useLocation();
  const { tab } = new URLSearchParams(location.hash);
```

Reply:
@peripheralist could you remind us why we need to use `HashRouter` for the app? I recall their being a reason related to Fleek/IPFS...

Reply:
@johnnyd-eth I tested this one, it's gucci:

```js
  const location = useLocation()
  const params = new URLSearchParams(location.search)
  const trendingTabOpen = params.get('tab')
```

Reply:
![nice](https://user-images.githubusercontent.com/96150256/151907732-8ff8ebf0-def7-4f6b-92b5-5b028421605e.gif)

Beautiful got him 🔨

Reply:
@aeolianeth  i believe the HashRouter is necessary to parse the /#/ routes...

but, this was one of those things developed ages ago that i don't remember fully. might not be necessary